app/uk/gov/hmrc/fileupload/controllers/FileUploadController.scala

```diff
@@ -39,12 +41,22 @@ class FileUploadController(withValidEnvelope: WithValidEnvelope,
                            now: () => Long)
                           (implicit executionContext: ExecutionContext) extends Controller {
 
-  def uploadWithEnvelopeValidation(envelopeId: EnvelopeId, fileId: FileId) =
+  val helper = new RedirectionFeature(Seq()) // FIXME configuration
```

Comment:
Prob a minor thing. But not a fan of having generically named val's called things like _helper_ esp if it's a specific to a feature of the class.